README.md

```diff
@@ -57,6 +57,28 @@ exports.onRouteChange = () => {
 }
 ```
 
+### Customize the scrolling behavior
+```js
```

Comment:
We should document that you must return `true` if the scroll was successful.

Reply:
Will add that here.